src/services/ui/src/features/submission/waiver/capitated/capitated-1915-b-waiver-amendment.tsx

```diff
@@ -129,7 +126,7 @@ export const Capitated1915BWaiverAmendmentPage = () => {
 
   return (
     <SimplePageContainer>
-      <BreadCrumbs options={formCrumbsFromPath(location.pathname)} />
+      {!id && <BreadCrumbs options={formCrumbsFromPath(location.pathname)} />}
```

Comment:
ActionWrapper already does crumbs for this as an action, so in lieu of an ID path parameter (meaning arriving via New Submission) we'll show our own.